src/profiles.jl

```diff
@@ -36,24 +45,26 @@ function Base.getindex(
 end
 
 """
-    StrategicProfile(vals)
+    StrategicProfile(vals::Vector{P}) where {T<:Duration, P<:TimeProfile{T}}
+    StrategicProfile(vals::Vector{<:Duration})
 
 Time profile with a separate time profile for each strategic period.
+If the input is a
```

Comment:
Incomplete ?

Reply:
Yeah, I thought of adding a statement regarding the input, but realized it is sufficient to check the function calls in the docstring.